notebooks/community/model_garden/model_garden_pytorch_flan_t5_xxl.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<p>Combine the 2 text boxes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1961/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
done